main/inc/lib/template.lib.php

```diff
@@ -805,7 +805,7 @@ private function set_header_parameters()
         $this->assign('text_direction', api_get_text_direction());
         $this->assign('section_name', 'section-'.$this_section);
 
-        $favico = '<link rel="shortcut icon" href="'.api_get_path(WEB_PATH).'favicon.ico" type="image/x-icon" />';
+        $favico = '<link rel="shortcut icon" href="'.api_get_path(WEB_CSS_PATH). 'themes/' .$this->theme. '/images/favicon.ico" type="image/x-icon" />';
```

Comment:
Its done !! i Added the Verification to the default favicon and the themed favicon